### PR TITLE
Create Recipe App with Rails

### PR DESCRIPTION
**Pull request by alonsofl**
## Summary
Created a Recipe App with Rails that allow users to add food(ingredients) and recipes. This app uses authorization to delete and update each recipe and/or food(ingredients).

#### Details
- Installed Ruby on Rails framework.
- Created migration files according to the [ERD Diagram](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/recipe_app_erd.png)
- Added the required routes to recipes, foods, shopping lists, and forms to register, login, and add items.
- Configured controllers to handle requests.
- Added the required models and validations.
- Added Devise, CanCanCan, RSpec, and Capybara gems.
- Added the required views for each controller and route.
- Secured the app from n+1 problems.
- Added testing to the project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,91 @@
...
+- [Rails - Has Many Method](https://api.rubyonrails.org/v6.1.4/classes/ActiveRecord/Associations/ClassMethods.html#method-i-has_many)
+- [Rails - Belongs To Method](https://api.rubyonrails.org/v6.1.4/classes/ActiveRecord/Associations/ClassMethods.html#method-i-belongs_to)
+- [URL Helpers](https://api.rubyonrails.org/v5.2.3/classes/ActionView/Helpers/UrlHelper.html)
+- [Form Helpers](https://guides.rubyonrails.org/form_helpers.html)
+
+## Author
+
+**Abimbola Buari**
+- GitHub - [@alonsofl](https://github.com/bimbolabuari)
+- LinkedIn - [Alonso Falconi](https://www.linkedin.com/in/bimbolabuari/)
```
Comment (DeliceLydia) [inline README.md:84]: - [ ] Add the right author account

Comment (DeliceLydia) [reply README.md:84]: - [ ] Please add a live demo in your README

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [ ] Because this project is collaborative you should all participate I can see that all commits were done by one person this is not professional because collaborating will prepare you in your future endeavors, please next time don't do this

### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (alonsofl) [reply README.md]: Thanks for your review! The suggestions have been addressed :smiley:

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

To Highlight: 
- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashboard. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Comment by alonsofl** [comment]
Thanks, guys! :raised_hands:

**Pull request merged**